src/components/circular-progress/circular-progress.tsx

```diff
@@ -83,11 +89,19 @@ export class CircularProgress {
                 aria-valuenow={this.value}
                 style={{ '--percentage': currentPercentage }}
             >
-                <span class="value">
-                    {abbreviate(value)}
-                    <span class="suffix">{this.suffix}</span>
-                </span>
+                <div class="prefix-value">
```

Comment:
adding this deep nesting doesn't seem necessary. I think we can create a nice layout with the need of complications in the HTML.